test/metabase/api/common/internal_test.clj

```diff
@@ -123,30 +136,40 @@
               (handler req)
               (catch Exception e (mw.exceptions/api-exception-response e)))))
 
-(deftest defendpoint-query-params-test
+(defn- do-with-jetty-server [f]
```

Comment:
Did a little bit of test cleanup in this namespace -- big thing I noticed was that a few of these tests created a new Jetty server on a random port and then never stopped that server. Reworked the tests to use a new `with-jetty-server` macro that shuts down the server when we're done with it